ui/desktop/electron-app/package.json

```diff
@@ -26,6 +26,7 @@
     "electron-is-dev": "^2.0.0",
     "electron-log": "^5.1.2",
     "electron-squirrel-startup": "^1.0.0",
+    "electron-store": "^8.2.0",
```

Comment:
I went with `electron-store` as it seemed small with minimal dependencies and a decent amount of usage compared to other packages. It also under the hood just uses the `app.getPath` [userData](https://www.electronjs.org/docs/latest/api/app#appgetpathname) to store the config files.